mvc/middleware.js

```diff
@@ -0,0 +1,8 @@
+exports.handle404Errors = (req, res, next) => {
+  res.status(404).send({ msg: "endpoint does not exist" });
+};
+
+exports.handleServerErrors = (err, req, res, next) => {
+  res.status(500).send({ msg: "Internal server error" });
```

Comment:
I would also log the error in here so that you can debug it and handle it better.